apps/frontend/src/libs/components/avatar/avatar.tsx

```diff
@@ -0,0 +1,35 @@
+import {
+	useAppDispatch,
+	useAppSelector,
+	useEffect,
+} from "~/libs/hooks/hooks.js";
+import { actions as userActions } from "~/modules/users/users.js";
+
+import { Loader } from "../components.js";
+import styles from "./styles.module.css";
+
+const Avatar = (): JSX.Element => {
+	const dispatch = useAppDispatch();
+	const users = useAppSelector(({ users }) => users.users);
+	const dataStatus = useAppSelector(({ users }) => users.dataStatus);
+	const isLoading = dataStatus === "pending";
+	const ZERO = 0;
+
+	useEffect(() => {
+		void dispatch(userActions.loadAll());
+	}, [dispatch]);
+
+	return (
+		<div className={styles["avatar"]}>
+			{isLoading ? (
+				<Loader />
+			) : (
```

Comment:
We don't need to display loader here, this element won't be used in places where there is no user yet
```suggestion
```